.gitmodules

```diff
@@ -9,4 +9,12 @@
 [submodule "ocaml-dockerfile"]
 	path = ocaml-dockerfile
 	url = https://github.com/ocurrent/ocaml-dockerfile.git
-	branch = master
\ No newline at end of file
+	branch = master
+[submodule "ocaml-ci"]
+	path = ocaml-ci
+	url = https://github.com/ocurrent/ocaml-ci.git
+	branch = master
+[submodule "ocaml-version"]
+	path = ocaml-version
+	url = https://github.com/ocurrent/ocaml-version.git
+	branch = master
```

Comment:
You should be able to depend on the opam version of this package. I am not sure we want to bring in `ocaml-ci` as this will create an import loop. Can the problem be broken down into a smaller reproduction case?

Reply:
You right, I don't how I missed that.